CONTRIBUTING.md

```diff
@@ -140,6 +140,26 @@ Note: For a list of available gulp commands use:
 
 `$ gulp`
 
+#### Trouble shooting(For Mac)
+
+If you have problem serve the site locally after all these steps, please try the following steps:
```

Comment:
Change to:

If you have a problem serving the site locally, please try the following: